ext/TOML/src/TOML.jl

```diff
@@ -3,6 +3,8 @@
 module TOML
     using Dates
 
+    const TYPE = Union{AbstractDict,AbstractArray,AbstractString,DateTime}
```

Comment:
@00vareladavid: add `Bool` here; also maybe move it into the `print.jl` file so it's close to the other related code. This kind of seems like a bad place to have put this definition in hind sight.